desktop/cmp/dash/canvas/impl/DashCanvasView.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {hoistCmp, ModelPublishMode, uses} from '@xh/hoist/core';
+import {contextMenu} from '@xh/hoist/desktop/cmp/contextmenu';
+import {elementFromContent} from '@xh/hoist/utils/react';
+import {Icon} from '@xh/hoist/icon';
+import {panel} from '../../../panel';
+import {button} from '../../../button';
+import {popover, Position} from '@xh/hoist/kit/blueprint';
+import {DashCanvasViewModel} from '../DashCanvasViewModel';
+
+/**
+ * Implementation component to show an item within a DashCanvas.  This component
+ * is used by DashCanvas's internal implementation to:
+ *
+ *   - Mount/unmount its contents according to `DashCanvasViewSpec.renderMode`.
+ *   - Track and trigger refreshes according to `DashCanvasViewSpec.refreshMode`.
+ *   - Stretch its contents using a flex layout.
+ *
+ * @private
+ * @Beta
```

Comment:
Don't think @Beta is real, and in any case not needed for this if its private anyway.